Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java

```diff
@@ -46,6 +50,22 @@ public RandomCutForestState convert(String json, Precision precision) throws IOE
         return convert(forest, precision);
     }
 
+    public Optional<RandomCutForestState> convert(ArrayList<String> jsons, int numberOfTrees, Precision precision)
```

Comment:
a javadoc would be useful such as on when the result will be empty.

Reply:
This code is run once and not intended for general use.

Reply:
ok. The data type for jsons should be `List<String>`

Reply:
"not intended for general use" -> this is to spec of what the caller suggested.